README.md

```diff
@@ -49,7 +49,7 @@ app.post('/graphql', graphqlHTTP({
 
 app.get('/graphql', graphqlHTTP({
   schema: MyGraphQLSchema,
-  graphiql: true // set it to false to disable the GraphiQL interface
```

Comment:
I see your point and I don't seek to continue this conversation nor to push to re-introduce the comment.
The difference is that due to the nature of Restify you are required to setup GraphQL on multiple methods, the GET and the POST (if you need both, for example personally I'm ok with just the POST), this might cause confusion with the options that you pass in one or the other so I thought being a little bit more explicit could help and wouldn't harm.